app.py

```diff
@@ -33,7 +33,7 @@ def create_response(data=None, message=None, status=200):
 # create a test route
 @app.route('/test', methods=['GET'])
 def test():
-  return make_response(jsonify({'message': 'test route'}), 200)
```

Comment:
i would prefer it to be this way, standard practices maybe 😅

Reply:
cool!